types/mongodb/index.d.ts

```diff
@@ -360,6 +360,12 @@ export interface MongoClientOptions extends
      * Mechanism for authentication: DEFAULT, GSSAPI, PLAIN, MONGODB-X509, 'MONGODB-CR', SCRAM-SHA-1 or SCRAM-SHA-256
      */
     authMechanism?: 'DEFAULT' | 'GSSAPI' | 'PLAIN' | 'MONGODB-X509' | 'MONGODB-CR' | 'SCRAM-SHA-1' | 'SCRAM-SHA-256' | string;
+
+    /** Type of compression to use */
+    compression?: {
+        /** The selected compressors in preference order */
+        compressors?: Array<('zstd' | 'snappy' | 'zlib')>;
```

Comment:
Is there any documentation regarding `'zstd'`.
http://mongodb.github.io/node-mongodb-native/3.6/api/MongoClient.html#.connect list only the other two

Reply:
@CaselIT yes in the global mongodb doc : https://docs.mongodb.com/manual/core/wiredtiger/#compression

It's a high compression / low CPU mode added in mongo 4.2.

The node-mongodb-native documentation is far from up to date :/

Reply:
Are you sure that the Node.js driver supports `zstd` though? 🤔 

Doesn't seem like it?
https://github.com/mongodb/node-mongodb-native/search?q=zstd&unscoped_q=zstd

Reply:
hummmm, you're right.
Well seen, I'll remove it from the list

Reply:
done

Reply:
maybe an issue could be opened in the nodejs driver repo?